t/bad_subclass.t

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env perl
```

Comment:
Tests should use strict & warnings. I updated all the test in master to start with that.